tests/Test/BlackBoxTest.php

```diff
@@ -45,20 +52,20 @@ public function gaugesShouldBeOverwritten()
         $metricsResult = $this->client->get('/examples/metrics.php?adapter=' . $this->adapter);
         $body = (string)$metricsResult->getBody();
         echo "\nbody: " . $body . "\n";
-        $this->assertThat(
+        self::assertThat(
```

Comment:
Why the change from `$this->` to `self::`?

Reply:
so, while Sebastian has no preference, the functions are all static, so PHPStan will complain about an instance invocation instead of a static invocation. For these reasons most projects I have worked on resort to using `self::`. So I just instinctively changed it, do you have a preference for this?

Reply:
Nope i'm fine with the static approach too, even if it is a bit strange in the first place.